test/test_train_network.py

```diff
@@ -122,7 +122,7 @@ def teardown_class(cls):
                 print("Could not clean up {} - you may want to remove it manually.".format(modelPath))
 
     # --------------train_model tests-------------- #
-    @pytest.mark.integration
+    @pytest.mark.debug
```

Comment:
Do we need to change the marker of `test_train_network`.py from `integration` to `debug`?

Reply:
Yes i'll change it, thanks for catching it